eng/pipelines/common/templates/pipeline-with-resources.yml

```diff
@@ -59,6 +69,12 @@ resources:
       env:
         ROOTFS_DIR: /crossrootfs/ppc64le
 
+    - container: debian-11-gcc12-amd64
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:debian-11-gcc12-amd64
```

Comment:
Not in this PR, but can we consider moving this image to be based off Ubuntu 22.04 instead of Debian 11 (Debian 11 has an older CMake version than the one we want to use and IIRC I don't think we want to use the Kitware feed in our builds)?